src/webview/mod.rs

```diff
@@ -413,6 +413,13 @@ impl Drop for WebView {
   }
 }
 
+impl Deref for WebView {
+  type Target = Window;
```

Comment:
I think this is not intuitive.
But is there something context?

Reply:
manipulating the window after the webview has been added it to it requires calling `.window()` method on the webview, so this is just for convenience and since the webivew and the window is 1:1 related, it should be ok.

Reply:
Is exporting different type used usually?
I have not ever seen this pattern.
But I think it is make sense if there is usecase for user.

Reply:
> Is exporting different type used usually?

Yes, it is actually a common pattern, some examples in rust std: `Arc`, `Rc`, `MutexGuard` and `Box`, they all implement `Deref` or some times `DerefMut` into the inner type.

Reply:
Sure. But although these types have inner type explicitly like T of Rc<T>, webview type has window as struct field.
These are only a little difference, so it may not be carefully😅

Reply:
There's an issue about multiple webview in one window. Do we decide it's going to be one webview one window forever?

Reply:
This only for current implementation, once we implement multiple webview per window, we can reevaluate this.

Reply:
I'm sorry but I still don't think this is a good practice. `Deref` trait is used for dereferencing. `Window` and `WebView` are still two different types IMHO.
I think in prepare of multi webview in the future, we could probably propose a `WebViewWindow` type and it then has valid point to use `Deref` to `Widnow`.

Reply:
sounds reaasonable